src/sinks/util/adaptive_concurrency/service.rs

```diff
@@ -275,10 +275,10 @@ mod tests {
 
         let in_flight = stats.in_flight.stats().unwrap();
         assert_eq!(in_flight.max, 1);
-        assert_eq!(in_flight.mean, 1.0);
+        approx::assert_relative_eq!(in_flight.mean, 1.0);
 
         let observed_rtt = stats.observed_rtt.stats().unwrap();
-        assert_eq!(observed_rtt.mean, 1.0);
+        approx::assert_relative_eq!(observed_rtt.mean, 1.0);
```

Comment:
This *should* also be exact due to the clock being frozen before the test, but I could see some error creeping in, so this makes more sense.

Reply:
The behavior change is due to tokio 1.5 -> 1.6 upgrade as I noted over here: https://github.com/timberio/vector/pull/7477#issuecomment-843337497

I was hoping you might take a look at it when you get some time @bruceg 😄

Reply:
Hmm, I missed seeing that comment. I'll look.

Reply:
Cool, I'll close this out then.